internal/topic/topicreadercommon/committer.go

```diff
@@ -148,6 +148,7 @@ func (c *Committer) pushCommitsLoop(ctx context.Context) {
 
 		onDone := trace.TopicOnReaderSendCommitMessage(
 			c.tracer,
+			&ctx,
```

Comment:
Here and in other place. 
Don't transfer pointer to owned context directly - you shold copy local context, pass pointer to copy, them copy result to local context.

Pointer to context may be stored and then context can be changed by other code and data race may be happen.

[example](https://github.com/ydb-platform/ydb-go-sdk/blob/b035f5d08e59ef5480efcf31b9eae4171c253250/internal/topic/topicwriterinternal/writer_transaction.go#L34-L43)